packages/babel-plugin-transform-react-display-name/README.md

```diff
@@ -7,15 +7,19 @@
 **In**
 
 ```js
-var foo = React.createClass({});
+var foo = React.createClass({}); // React <= 15
```

Comment:
Sure thing fellas. Sorry it took me a bit to circle back here. Got pulled away on some react-native stuff. Wording has been updated.